src/oauth-provider.ts

```diff
@@ -780,6 +780,19 @@ export class OAuthProvider {
   fetch(request: Request, env: any, ctx: ExecutionContext): Promise<Response> {
     return this.#impl.fetch(request, env, ctx);
   }
+
+  /**
+   * Gets OAuthHelpers for the given environment, setting env.OAUTH_PROVIDER if not already set
+   * This is a getter property that returns a function, allowing it to be accessed like a normal class member
+   * without being exposed as an RPC method.
```

Comment:
Hate to say but getters can, in fact, be invoked over RPC. It's an explicit feature of the RPC system.

You could perhaps create a separate freestanding function `getOAuthHelpers(provider :OAuthProvider)`, and this function could work by reading a property whose name is a private `Symbol`.